tests/tape/test_tape.py

```diff
@@ -685,6 +685,19 @@ def make_tape(self):
 
         return tape, params
 
+    @pytest.fixture
+    def make_tape_symbolic(self):
+        params = [[], [0.432], 0.123, [[], [0.546]], []]
+
+        with QuantumTape() as tape:
```

Comment:
we're avoiding using `with tape` internally. Can you put this fixture+test in `test_qscript.py` instead and use QuantumScripts?

Reply:
+1 we should be testing this on `qscript` instead, because this is not queuing specific behaviour.